result/brace-style.js

```diff
@@ -0,0 +1,209 @@
+if (foo) {
+  bar();
+} else {
```

Comment:
__[development][eslint]__
* Error - Closing curly brace appears on the same line as the subsequent block. (brace-style)